spm/doc/INTRO.md

```diff
@@ -0,0 +1,117 @@
+
+#### PSA Levels of Isolation
+
+If you are prototyping software or using platforms without SAU or MPU, you can choose to have no isolation between the SPE and NSPE (sometimes referred to as Level 0), although this is not specified by the PSA. However, for production software, you should consider implementing one of the following levels of isolation:
+
+* **Level 1 - SPE isolation** In this level, the SPE is fully isolated from the non-secure application firmware and hardware.
+* **Level 2 - Root of Trust isolation** In this level, the SPE is fully isolated from the non-secure application firmware and hardware and the Trusted Partitions (Secure Partitions that implement Root of Trust services) are isolated from other Secure Partitions.
+* **Level 3 - Maximum firmware isolation** In this level, the SPE is fully isolated from the non-secure application firmware and hardware, and all Secure and Trusted Partitions are individually sandboxed from each other and from the SPM.
+
+### Using Secure Partitions
+
+Secure partitions are located within the SPE, and must contain at least one set of related security operations (known as a Secure Function) or at least one Interrupt Request (IRQ). You can have multiple Secure Functions in a single Secure Partition.
+
+For a Secure Partition, you need:
+
+* The **Secure Partition code**, which must:
+  * Be single threaded.
+  * Be structured as a loop that waits for inputs.
+  * Never exit its loop (considered as a programming error).
+  * Be written in C or 'extern "C"' to avoid C++ name mangling. 
+  * Follow PSA IPC rules. Secure partitions communicate with each other using the IPC API defined in [IPC API](https://github.com/ARMmbed/PSA-IPC-doc/blob/master/IPC_revision.md). All IPC messages must eventually be completed [`call psa_end()`]. Note that the SPM does not schedule IPC messages fairly.
+* A **manifest file** in JSON format, that describes the Secure Partition characteristics. The specifications in the manifest file are validated during the build process and at run time.
+
+### Manifest File Example
+
+The Secure Partition manifest file describes the properties of the Secure Partitions. In this file:
+
+* **entry_point** is the function name of the Partition's thread.
+* **source_files** is the list of source files containing the Partition's code.
+* **heap_size** sets the heap size for platforms that have an isolation level of 2 and higher.
+* **secure_functions** is the list of the Partition's Secure Functions with their properties.
+* **extern_sfids** defines a dependency to other Secure Function (referenced by SFID).  If access between a Partition (acting as client) and a Secure Function (acting as server) is not specified in the manifest then the client will not be able to send any messages to the Secure Function.
```

Comment:
Please change the second sentence to "If the manifest does not specify the access between a partition (acting as client) and a secure function (acting as server), then the client is not be able to send any messages to the secure function."